docs/content/docs/dev/datastream/fault-tolerance/checkpointing.md

```diff
@@ -292,4 +292,25 @@ The final checkpoint would be triggered immediately after all operators have rea
 without waiting for periodic triggering, but the job will need to wait for this final checkpoint 
 to be completed.
 
+## Unify file merging mechanism for checkpoints
+
+Flink 1.20 introduces a unified file merging mechanism for checkpointing, which allows scattered small checkpoint files 
+to be written into a single file, reducing the number of file creations and file deletions, helping to alleviate the pressure
+of file system metadata management and file flooding problem. The unified fie merging mechanism can be enabled by setting
+the property `state.checkpoints.file-merging.enabled` to `true`. **Note** that enabling this mechanism may lead to space amplification,
+that is, the actual occupation on the file system will be larger than the checkpoint size. `state.checkpoints.file-merging.max-space-amplification` 
```

Comment:
IIUC, the metric of checkpoint size should be consistent, right ?
If It's compared with before or acutal state, let's adjust it as `before checkpoint size` or `actual state size`.

Reply:
Right, thanks for the suggestion, I adjusted it as `actual state size`.